doc/fluxing.rst

```diff
@@ -32,6 +32,8 @@ science object's counts per second to yield a fluxed science
 spectrum.
 
 The sensitivity function is written to disk as a FITS file.
+This function converts the extracted count spectrum
+to f_lambda units (erg/s/cm^2/Ang) scaled by 1e-17.
```

Comment:
Why "scaled by"?  Why not "F_lambda units of 1e-17 erg/s/cm^2/angstrom"?

Astronomy should invent a unit that is 1e-17 erg so that we don't have to keep writing it like this!

Reply:
edited